Library/Homebrew/cmd/update-report.rb

```diff
@@ -293,8 +294,6 @@ def link_completions_manpages_and_docs(repository = HOMEBREW_REPOSITORY)
   end
 
   def migrate_gcc_dependents_if_needed
```

Comment:
Might be worth moving the whole method into `os/linux/cmd/update-report` and have just an empty one here?